compiler/rustc_mir_transform/src/deduce_param_attrs.rs

```diff
@@ -169,20 +168,9 @@ pub(super) fn deduced_param_attrs<'tcx>(
     // see [1].
     //
     // [1]: https://github.com/rust-lang/rust/pull/103172#discussion_r999139997
```

Comment:
The FIXME above is resolved. Would be also nice to describe that "contains no UnsafeCells" is now checked during ABI adjustment.